client/index.html

```diff
@@ -9,15 +9,25 @@
     <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.css">
     <link rel="stylesheet" href="css/healthChart.css">
     <link rel="stylesheet" href="css/main.css">
+	<link rel="stylesheet" href="bower_components/nouislider/jquery.nouislider.css" />
 
+	<script src="bower_components/jquery/dist/jquery.js"></script>
     <script src="bower_components/angular/angular.js"></script>
     <script src="bower_components/angular-ui-router/release/angular-ui-router.js"></script>
     <script src="bower_components/moment/moment.js"></script>
+	<script src="bower_components/nouislider/jquery.nouislider.js"></script>
+	<script src="bower_components/nouislider/Link.js"></script>
+	<script src="bower_components/angular-nouislider/src/nouislider.js"></script>
+
 
     <script src="js/nug-skeleton-app.js"></script>
     <script src="js/dashboardCtrl.js"></script>
     <script src="js/settingsCtrl.js"></script>
+<<<<<<< HEAD
```

Comment:
Oops, these are merge conflict tags still in the file.  Make sure to get rid of <<<<< ====== and >>>>>>